auth/signup.spec.js

```diff
@@ -0,0 +1,86 @@
+process.env.NODE_ENV = 'test'
+
+var chai = require('chai')
+var chaiHttp = require('chai-http')
+var server = require('../index')
+var cleandb = require('../utils/cleandb')
+var should = chai.should()
+
+chai.use(chaiHttp)
+
+describe('Auth signup tests', function() {
+
+  beforeEach(function() {
+    return cleandb()
+  })
+
+  it('should return error due the one empty field (birthday)', function(done) {
```

Comment:
are you sure that `done` is needed here? Wouldn't it be sufficient with just returning the promise chain? like
```
return chai.request(server)
  .post('/auth/signup')
  // ...
```
?